lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err) {});
+ * myFile.acl.makePrivate(function(err) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
+};
+
+/**
+ * Make a {module:storage/bucket} or {module:storage/file} object publicly
+ * readable.
+ *
+ * This is a short-hand method which will grant "reader" permissions to the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePublic
+ *
+ * @example
+ * myBucket.acl.makePublic(function(err, aclObject) {});
```

Comment:
What confused you, that it was private documentation? It's marked `@private` to keep it out of the published docs on the site. I could add the `/*! Developer Documentation` heading, though that's typically for broader documentation purposes, such as "The methods that follow are for...". This is just the documentation for this class.

If the confusion was `acl` vs `aclObject`, were you confused about the object itself, or that `acl` in this example wouldn't be the same object as `aclObject` from another example?